docs/userGuide/deployingTheSite.md

```diff
@@ -52,18 +52,44 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 ### Deploying to GitHub Pages via Travis CI
 **You can setup [<tooltip content="a platform for Continuous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
 
-Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+#### Adding your repository to Travis CI
+
+<panel header="{{icon_info}} Travis CI Migration" type="info" expanded>
+
+As of May 2018, Travis CI was [undergoing migration to `travis-ci.com`](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/), which changes the way you setup repositories on Travis CI. 
+
+- If you are new to Travis CI, or adding a new repository, you should **add your repository on `travis-ci.com`**.
+- If you are adding an existing repository, you should **add your repository on `travis-ci.org`**.
+</panel>
```

Comment:
A `<p/>` here to add some whitespace between the panel and the tabs below?

Reply:
Styles like this should use a class (e.g. `pb-1` from [Bootstrap 4's Spacing Utilities](https://getbootstrap.com/docs/4.1/utilities/spacing/)) instead.

Reply:
True. If we look at this as from a user's PoV, the requirement is simple, and the solution is disproportionately complex. Ideally, the user shouldn't even have to specifically add space here (it should have been added by default). Is there a way to improve this case? In particular, I find myself adding `<p/>` after the last panel whenever I have consecutive panels.

Reply:
Is there a live example?

Reply:
MarkBind doesn't add padding below a panel, which is the right thing to do because when we have multiple consecutive panels, we don't want too much space _between_ panels. However, we don't want normal content below a panel to stick too close to the panel bottom either. That's where the author is forced to add some whitespace below a panel. In my case I use `<p/>` as it is the easiest way. It is nice if MarkBind can add padding between panel bottom and non-panel contents immediately below it.

Reply:
Is there a live example on a website?

Reply:
Added a `<p/>` as adding a `pb-*` class to `<panel>` does not seem to work out of the box.

Reply:
@Xenonym That's because a `<panel>` is a component that's converted to a `<span class="card-container">`.

Reply:
@acjh the class *is* added if you specify `<panel class="pb-3">`, it just doesn't seem to have an effect for some reason:

![pb-3 class on panel](https://user-images.githubusercontent.com/1782590/52912614-97afd900-32ee-11e9-8dd5-8945947408cc.png)

Reply:
Try `mb-3`.

Reply:
@acjh `mb-3` has a similar result; the class is applied but there is no visible difference.

Reply:
Oh, because it's an inline `<span>`. Need to apply it to the inner `.card`, which may involve a CSS rule in the stylesheet instead of specifying a class on the `<panel>`.

We can proceed with `<p/> first.